core/domain/stats_jobs_one_off.py

```diff
@@ -522,7 +536,12 @@ def reduce(exp_id, values):
         event_dict_idx = 0
         event_dict = sorted_events_dicts[event_dict_idx]
         for version in versions:
-            datastore_stats_for_version = old_stats[version - 1]
+            datastore_stats_for_version = copy.deepcopy(old_stats[version - 1])
+            if datastore_stats_for_version is None:
```

Comment:
This guard conditional could be here for safety, but ideally shouldn't be happening. Our Regenerate job should be filling up all models.